### PR TITLE
 - version 0.3.1

### PR DESCRIPTION
**Pull request by oualib**
- Simplifying Connection Code
 - Simplifying Displaying Options
 - Correcting LinearRegression, Ridge, ElasticNet, Lasso, TSA
 - Adding WOE/IV
 - New sample method
 - Correcting kendall Tau
 - Adding/Correcting Unitary Tests

### CODE REVIEW

File: verticapy/tests/vModel/test_ridge.py [hunk 1]
```diff
@@ -63,9 +64,9 @@ def test_features_importance(self, model):
         fim = model.features_importance()
 
         assert fim["index"] == ['alcohol', 'residual_sugar', 'citric_acid']
-        assert fim["importance"] == [65.86, 33.35, 0.79]
+        assert fim["importance"] == [52.3, 32.63, 15.07]
         # TODO: it is nicer not to have Decimal for sign
```
Comment (afard) [inline verticapy/tests/vModel/test_ridge.py:68]: You've done this TODO comment, so you can remove it.

**Review by afard — APPROVED** [review]

**Comment by afard** [comment]
I'm surprised why your license/cla is not passed.

File: verticapy/tests/vModel/test_ridge.py (hunk 1 above)
Comment (oualib) [reply verticapy/tests/vModel/test_ridge.py]: Did it!

File: verticapy/connections/all/__init__.py [hunk 2]
```diff
@@ -1,46 +0,0 @@
-# (c) Copyright [2018-2020] Micro Focus or one of its affiliates.
```
[the remark this answers is not in the document]
Comment (oualib) [reply verticapy/connections/all/__init__.py:1]: I simplified DB connection using the ConfigParser

**Comment by oualib** [comment]
v0.3.1

**Pull request merged**